src/EFCore.PG/Extensions/NpgsqlNetworkExtensions.cs

```diff
@@ -0,0 +1,1180 @@
+        /// </exception>
+        public static bool LessThanOrEqual([CanBeNull] this DbFunctions _, IPAddress inet, IPAddress other)
+            => throw ClientEvaluationNotSupportedException();
+
+        /// <summary>
+        /// Determines whether an (IPAddress Address, int Subnet) is less than or equal to another (IPAddress Address, int Subnet).
+        /// </summary>
+        /// <param name="_">The <see cref="DbFunctions"/> instance.</param>
+        /// <param name="cidr">The left-hand cidr.</param>
+        /// <param name="other">The right-hand cidr.</param>
+        /// <returns>
+        /// True if the (IPAddress Address, int Subnet) is less than or equal to the other (IPAddress Address, int Subnet); otherwise, false.
+        /// </returns>
+        /// <exception cref="NotSupportedException">
+        /// This method is only intended for use via SQL translation as part of an EF Core LINQ query.
+        /// </exception>
+        public static bool LessThanOrEqual([CanBeNull] this DbFunctions _, (IPAddress Address, int Subnet) cidr, (IPAddress Address, int Subnet) other)
+            => throw ClientEvaluationNotSupportedException();
+
+        /// <summary>
+        /// Determines whether an <see cref="PhysicalAddress"/> is less than or equal to another <see cref="PhysicalAddress"/>.
+        /// </summary>
+        /// <param name="_">The <see cref="DbFunctions"/> instance.</param>
+        /// <param name="macaddr">The left-hand macaddr.</param>
+        /// <param name="other">The right-hand macaddr.</param>
+        /// <returns>
+        /// True if the <see cref="PhysicalAddress"/> is less than or equal to the other <see cref="PhysicalAddress"/>; otherwise, false.
+        /// </returns>
+        /// <exception cref="NotSupportedException">
+        /// This method is only intended for use via SQL translation as part of an EF Core LINQ query.
+        /// </exception>
+        public static bool LessThanOrEqual([CanBeNull] this DbFunctions _, PhysicalAddress macaddr, PhysicalAddress other)
+            => throw ClientEvaluationNotSupportedException();
+
+        /// <summary>
+        /// Determines whether an <see cref="IPAddress"/> is equal to another <see cref="IPAddress"/>.
+        /// </summary>
+        /// <param name="_">The <see cref="DbFunctions"/> instance.</param>
+        /// <param name="inet">The left-hand inet.</param>
+        /// <param name="other">The right-hand inet.</param>
+        /// <returns>
+        /// True if the <see cref="IPAddress"/> is equal to the other <see cref="IPAddress"/>; otherwise, false.
+        /// </returns>
+        /// <exception cref="NotSupportedException">
+        /// This method is only intended for use via SQL translation as part of an EF Core LINQ query.
+        /// </exception>
+        public static bool Equal([CanBeNull] this DbFunctions _, IPAddress inet, IPAddress other)
```

Comment:
Thinking about this again, this isn't necessary... EF Core already translates `x.Equals(y)` to `x = y` in SQL.

So it should be fine to remove all `Equal` and `NotEqual` overloads below.